implementations/rust/ockam/ockam_command/src/relay/create.rs

```diff
@@ -50,6 +49,10 @@ pub struct CreateCommand {
     /// Authorized identity for secure channel connection
     #[arg(long, id = "AUTHORIZED")]
     pub authorized: Option<Identifier>,
+
+    /// Relay address to use. By default, inherits the relay name.
+    #[arg(long)]
+    relay_address: Option<String>,
```

Comment:
In what scenario is this useful? IIUC, this is to define the name of the relay that will be created while the alias is only used in the node to store it in the registry?

Reply:
By default the alias defines both the name of the relay and the address of the relay.
Normally, you would only use alias, but if you want to use the same address for different projects, multiple projects relaying toward the same node, then you need relay_address.
Another simple use case is to keep compatibility with a previously named address while renaming the relay itself.